admin_domains.php

```diff
@@ -599,6 +613,39 @@
 						'action' => 'su', 'id' => $customer['customerid'])) . '" rel="external">' . $customer['loginname'] . '</a>)';
 				}
 
+				$pubkey = $result['dkim_pubkey'];
```

Comment:
why so complicated, just use php-function like `openssl_pkey_get_details`

Reply:
I do not have the time to dig into this...